Formula/wxwidgets.rb

```diff
@@ -30,6 +30,7 @@ class Wxwidgets < Formula
     depends_on "gtk+3"
     depends_on "libsm"
     depends_on "mesa-glu"
+    depends_on "webkitgtk"
```

Comment:
`wxwidgets` doesn't seem to be linking with `webkitgtk`, despite `--enable-webviewwebkit` being passed to `configure`.

Reply:
Would be nice to have proper logs here. Linux is really uncooperative sometimes...

Reply:
Ah, I think I see the problem. `configure` looks for `webkit2gtk-4.0`, but we ship `webkit2gtk-4.1`.